olm/src/jsMain/kotlin/io/github/matrixkt/olm/InboundGroupSession.kt

```diff
@@ -0,0 +1,95 @@
+package io.github.matrixkt.olm
+
+
+public actual class InboundGroupSession private constructor(private val ptr: JsOlm.InboundGroupSession) {
+
+
+    public actual constructor(sessionKey: String): this(JsOlm.InboundGroupSession()) {
+        try {
+            ptr.create(sessionKey)
+        } catch (e: Exception) {
+            clear()
+            throw e
+        }
+    }
+
+    public actual fun clear () {
+        ptr.free();
+    }
+
+    /**
+     * Retrieve the base64-encoded identifier for this inbound group session.
+     * @return the session ID
+     */
+    public actual val sessionId: String get() = ptr.session_id()
+
+    /**
+     * Provides the first known index.
+     * @return the first known index.
+     */
+    public actual val firstKnownIndex: Long get() = ptr.first_known_index()
+
+    /**
+     * Tells if the session is verified.
+     * @return true if the session is verified
+     */
+    public actual val isVerified: Boolean
+        get()  {
+            throw NotImplementedError("is verified is not implemented in matrix-org/olm")
```

Comment:
Calling something like `olm_inbound_group_session_is_verified(ptr)` doesn't work?

Reply:
nope, its not exported through js

Reply:
https://gitlab.matrix.org/matrix-org/olm/-/blob/master/javascript/olm_inbound_group_session.js